app/database_etl/owid_ingestion_handler/owid_data_handler.py

```diff
@@ -93,6 +98,35 @@ def get_vaccinations_per_hundred(country_name: str, midpoint_date: datetime, ful
     return float(ret) if not ret.empty else None
 
 
+def get_vaccination_policy(country_name: str, midpoint_date: datetime):
+    if str(midpoint_date) == 'NaT': return None
+    country_id = get_country_code(country_name)
+    offset = OFFSETS['VACCINATIONS']
+    target_date = (midpoint_date + offset).strftime('%Y%m%d')
+
+    country_data = vaccine_policy_rollout_df.loc[vaccine_policy_rollout_df['CountryCode'] == country_id]
+    country_data['Date'] = country_data['Date'].apply(lambda x: str(x))
+    ret = country_data.loc[(country_data['Date'] == target_date) &
```

Comment:
You're casting two DTs to strings to compare them. Is that the best way to architect this, or is there an approach in the `datetime` module that would be better?

Reply:
country_data['Date'] is type int not type DT so im casting them both to strings to compare

Reply:
got it